python/graphstorm/eval/evaluator.py

```diff
@@ -382,6 +383,9 @@ def do_early_stop(self, val_score):
         self._val_perf_list.pop(0)
         self._val_perf_list.append(val_score)
 
+        if early_stop and get_rank() == 0:
+            logging.info("Early stop criterion %s satisfied. Training job stopping.", self._early_stop_strategy)
```

Comment:
Should we log the val_score  and val_perf_list @zhjwy9343 ?

Reply:
Yes, need to create different early stop messages for different criterion, and output the `val_perf_list` for reference.

Reply:
@RonaldBXu see above